frontend/public/lib/imageOperations.ts

```diff
@@ -83,13 +84,12 @@ export async function getCompressedJPG(file, maxSizeMB): Promise<string> {
       drawImageOnCanvas(image, canvas);
       canvas.toBlob(
         async function (blob) {
-          const options = {
-            maxSizeMB: maxSizeMB,
-            useWebWorker: true,
-          };
           try {
-            const compressedFile = await imageCompression(blob as File, options);
```

Comment:
Since this function now doesn't do anything any longer with compression the function name is not accurate any longer.